lxqt fails to build on 17.09

## Issue description

LXQT fails to builid with some CMake error about `Qt5X11Extras`.


### Steps to reproduce

* Add `desktopManager.lxqt.enable = true;` to `configuration.nix`.
* sudo nixos-rebuild switch


## Technical details

* System: 17.09beta285.b50193f0ca (Hummingbird)
* Nix version: nix-env (Nix) 1.11.14
* Nixpkgs version: 17.09beta285.b50193f0ca
* Sandboxing enabled: build-use-sandbox = false


<details>
<summary> Build log </summary>

```
λ sudo nixos-rebuild switch                                                                                                 ~
building Nix...
building the system configuration...
these derivations will be built:
  /nix/store/3zr54d7ksv84lzb7x44d3ndb1gzvw3d8-lxqt-session-0.11.1.drv
  /nix/store/4g1xai4km8pawd48rr3bc9lvs822nlp9-system-path.drv
  /nix/store/07yc6wiqzvyv1l559k8d3qjpvmrx1bdy-unit-systemd-fsck-.service.drv
  /nix/store/1iqxdnn1lbfdw2llcdha57d0s4azm2wg-unit-polkit.service.drv
  /nix/store/3p6mz86mxrnhq0jp8spyj2yjj9y5srvp-dbus-1.drv
  /nix/store/47awrxcx3d22b39c4gj4anhl8pw7srnx-set-environment.drv
  /nix/store/56406wx6h3pnpcfx63crpr0rwvjh5gyn-unit-dbus.service.drv
  /nix/store/5gfs1zwhdddqkr79s1klyicn76brc776-etc-zshenv.drv
  /nix/store/c4fyy4gwb1ddnz1kz89qw4izd4qkcikk-user-units.drv
  /nix/store/q9dzs5vpffh4gnb92i3cvbp6w1q9jw39-etc-profile.drv
  /nix/store/w8l0cvr4kkqh5vfd8sblqhka6knppx65-system-units.drv
  /nix/store/xgx8lw7qgc5sklqmsxjn49fqf7s9izrf-etc.drv
  /nix/store/dgyfb5snmqhhfny02qq832aa8lskljjs-nixos-system-nixos-17.09beta285.b50193f0ca.drv
these paths will be fetched (40.19 MiB download, 94.36 MiB unpacked):
  /nix/store/0qdbfgwmsmd1z32ckczp72n6mmkwrpvi-oxygen-icons5-5.37.0
  /nix/store/c1mzw27bn6mcb03xc9lr30b4gr5j776p-gvfs-1.30.1
  /nix/store/q878rdkzgg41rfvfcbzpq3q7wxib8ml7-samba-4.6.4
fetching path ‘/nix/store/0qdbfgwmsmd1z32ckczp72n6mmkwrpvi-oxygen-icons5-5.37.0’...
fetching path ‘/nix/store/q878rdkzgg41rfvfcbzpq3q7wxib8ml7-samba-4.6.4’...
building path(s) ‘/nix/store/51cvzbhb3590jpjacl11i1kday12k1pn-lxqt-session-0.11.1’

*** Downloading ‘https://cache.nixos.org/nar/05lx2xi9aq9lj6y5gc15ikzikqcpnzsafy7ipf31bx06lwn1pc3q.nar.xz’ (signed by ‘cache.nixos.org-1’) to ‘/nix/store/0qdbfgwmsmd1z32ckczp72n6mmkwrpvi-oxygen-icons5-5.37.0’...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
*** Downloading ‘https://cache.nixos.org/nar/0p03gk47jvygijcl8nscc5dhlckgj54fz73lxlfm5cm0jya2ql9y.nar.xz’ (signed by ‘cache.nixos.org-1’) to ‘/nix/store/q878rdkzgg41rfvfcbzpq3q7wxib8ml7-samba-4.6.4’...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0unpacking sources
unpacking source archive /nix/store/drf15fw9886c31fs58g5bwy51mz27m8p-lxqt-session-0.11.1-src
source root is lxqt-session-0.11.1-src
patching sources
configuring
qmake: File exists
fixing cmake files...
cmake flags: -DCMAKE_BUILD_TYPE=Release -DCMAKE_SKIP_BUILD_RPATH=ON -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/51cvzbhb3590jpjacl11i1kday12k1pn-lxqt-session-0.11.1/include -DCMAKE_INSTALL_LIBDIR=/nix/store/51cvzbhb3590jpjacl11i1kday12k1pn-lxqt-session-0.11.1/lib -DCMAKE_INSTALL_NAME_DIR=/nix/store/51cvzbhb3590jpjacl11i1kday12k1pn-lxqt-session-0.11.1/lib -DCMAKE_INSTALL_PREFIX=/nix/store/51cvzbhb3590jpjacl11i1kday12k1pn-lxqt-session-0.11.1 -DPULL_TRANSLATIONS=NO 
-- The C compiler identification is GNU 6.4.0
-- The CXX compiler identification is GNU 6.4.0
-- Check for working C compiler: /nix/store/zgzak0grvimifrxr3yww1xbyix0qyrv8-gcc-wrapper-6.4.0/bin/gcc
-- Check for working C compiler: /nix/store/zgzak0grvimifrxr3yww1xbyix0qyrv8-gcc-wrapper-6.4.0/bin/gcc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
  0 29.5M    0 69259    0     0  69259      0  0:07:26  0:00:01  0:07:25 52668-- Detecting C compile features - done
-- Check for working CXX compiler: /nix/store/zgzak0grvimifrxr3yww1xbyix0qyrv8-gcc-wrapper-6.4.0/bin/g++
-- Check for working CXX compiler: /nix/store/zgzak0grvimifrxr3yww1xbyix0qyrv8-gcc-wrapper-6.4.0/bin/g++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Looking for XOpenDisplay in /nix/store/5sl8bywi0ybrwnrlqmzfh6d1jcvwbm4s-libX11-1.6.5/lib/libX11.so;/nix/store/rknkzqsrgmlvi6p5fa24szm1am0ln0nq-libXext-1.3.3/lib/libXext.so
  4 9835k    4  475k    0     0   237k      0  0:00:41  0:00:02  0:00:39  212k-- Looking for XOpenDisplay in /nix/store/5sl8bywi0ybrwnrlqmzfh6d1jcvwbm4s-libX11-1.6.5/lib/libX11.so;/nix/store/rknkzqsrgmlvi6p5fa24szm1am0ln0nq-libXext-1.3.3/lib/libXext.so - found
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for connect
-- Looking for connect - found
-- Looking for remove
-- Looking for remove - found
-- Looking for shmat
-- Looking for shmat - found
-- Found X11: /nix/store/5sl8bywi0ybrwnrlqmzfh6d1jcvwbm4s-libX11-1.6.5/lib/libX11.so
CMake Error at CMakeLists.txt:22 (find_package):
  By not providing "FindQt5X11Extras.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "Qt5X11Extras", but CMake did not find one.

  Could not find a package configuration file provided by "Qt5X11Extras" with
  any of the following names:

    Qt5X11ExtrasConfig.cmake
    qt5x11extras-config.cmake

  Add the installation prefix of "Qt5X11Extras" to CMAKE_PREFIX_PATH or set
  "Qt5X11Extras_DIR" to a directory containing one of the above files.  If
  "Qt5X11Extras" provides a separate development package or SDK, be sure it
  has been installed.


-- Configuring incomplete, errors occurred!
See also "/tmp/nix-build-lxqt-session-0.11.1.drv-0/lxqt-session-0.11.1-src/build/CMakeFiles/CMakeOutput.log".
builder for ‘/nix/store/3zr54d7ksv84lzb7x44d3ndb1gzvw3d8-lxqt-session-0.11.1.drv’ failed with exit code 1
killing process 11000
cannot build derivation ‘/nix/store/4g1xai4km8pawd48rr3bc9lvs822nlp9-system-path.drv’: 1 dependencies couldn't be built
killing process 11001
cannot build derivation ‘/nix/store/dgyfb5snmqhhfny02qq832aa8lskljjs-nixos-system-nixos-17.09beta285.b50193f0ca.drv’: 1 dependencies couldn't be built
error: build of ‘/nix/store/dgyfb5snmqhhfny02qq832aa8lskljjs-nixos-system-nixos-17.09beta285.b50193f0ca.drv’ failed
```
</details>

--- a/pkgs/desktops/lxqt/core/lxqt-session/default.nix
+++ b/pkgs/desktops/lxqt/core/lxqt-session/default.nix
@@ -1,4 +1,4 @@
-{ stdenv, fetchFromGitHub, cmake, pkgconfig, lxqt-build-tools, qtbase, qttools, qtsvg, kwindowsystem, liblxqt, libqtxdg, lxqt-common, xorg, xdg-user-dirs }:
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, lxqt-build-tools, qtbase, qttools, qtsvg, qtx11extras, kwindowsystem, liblxqt, libqtxdg, lxqt-common, xorg, xdg-user-dirs }:
 
 stdenv.mkDerivation rec {
   name = "${pname}-${version}";
@@ -22,6 +22,7 @@ stdenv.mkDerivation rec {
     qtbase
     qttools
     qtsvg
+    qtx11extras
     kwindowsystem
     liblxqt
     libqtxdg

